selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceHomePage.java

```diff
@@ -54,6 +59,48 @@ public void goToSpaceSettingTab() {
         }
     }
 
+  public void goToNewsTab() {
+    info("--Open News tab of the space");
+    $(ELEMENT_SPACE_NEWS_TAB).shouldBe(Condition.visible).click();
+    $(ELEMENT_SPACE_NEWS_CREATION_FORM).waitUntil(Condition.appears, Configuration.timeout);
+    info("News creation form is shown");
+  }
```

Comment:
News is not part of social, so we should avoid adding News stuff here if possible.